libs/surfaces/mackie/subview.cc

```diff
@@ -717,6 +718,23 @@ void SendsSubview::handle_vselect_event(uint32_t global_strip_position)
 	}
 }
 
+bool SendsSubview::handle_cursor_left_press()
+{
+	if (_current_bank >= 1)
+	{
+		_current_bank -= 1;
+	}
+	mcp().redisplay_subview_mode();
+
+	return true;
+}
+
+bool SendsSubview::handle_cursor_right_press()
```

Comment:
Does this allow to bank beyond the end of available tracks?

Reply:
Yes, you can keep banking right. That's how it is for the plugins subview, so I made it that way here too. I can put in a check to prevent that if you think it would be better.

Reply:
I think the plugin subview is wrong in this respect, and so ideally both would stop when they would otherwise bank beyond "the end".